.github/workflows/coverage.yml

```diff
@@ -0,0 +1,25 @@
+name: Tests and codecov
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: ['1.5']
+        julia-arch: [x64]
+        os: [ubuntu-latest]
```

Comment:
jeśli testujesz na jednym to nie potrzebujesz matrixa robić

Reply:
> powinien być chyba główny plik gdzie includuje się wszystkie testy - runtests.jl, tak widziałem na innych repach :D

A jest przecież dokładnie taki plik, z takim zadaniem xD [test/runtests.jl](https://github.com/Project-Summer-AI-Lab-Glider/nurse-scheduling-problem-solver/blob/TASK-coverage/test/runtests.jl)

Reply:
oki, niezauważyłem

Reply:
dobra z tym matrixem to pierdoła, dobra robota!